### DISCUSSION TITLE
Who Fixed the Alerts Scrolling?

### DISCUSSION THREAD

**Discussion by Acewiza** [Security-Onion-Solutions/securityonion#10619] [2.4]
Nice work.  No idea what or how, but acknowledging alerts sometimes seemed a bit iffy the way it jumped around on scrolling sometimes.  Was I on to something there, or just hallucinating again?

**Comment by dougburks** [comment]
I'm not sure what you're referring to.

**Comment by Acewiza** [comment]
I'm not either.  I noticed it right away processing alerts any time there's more than a page full.  Ack'ing alerts in the middle of the list caused a somewhat erratic scrolling behavior.  Unpredictable - sometimes scrolling up, sometimes down.  I didn't test it specifically in any detail - just something you notice.  It seems to scroll up always as expected now.

**Comment by Acewiza** [comment]
OK, plausible deniability, I get it...  ;-)

**Comment by Acewiza** [comment]
I can't be sure if some of the things I'm seeing in terms of the alerting aren't just the new onion being an order of magnitude faster, monitoring the same segment.

**Comment by Acewiza** [comment]
The hardware definitely has something to do with it, but not like I thought.  The new, faster beta machine is simply not seeing stuff the old onion is.  I'll swap interfaces or put a new NIC in it on the next rev...
